### PR TITLE
Add environments

### PR DESCRIPTION
**Pull request by KylieTims**
@PageUpPeopleOrg/redfins

### CODE REVIEW

**Comment by jackrust** [comment]
Everything looks good. 
**✓**

**Comment by KylieTims** [comment]
@jackrust @phdesign 
Even though tests were passing before they were not properly testing some functionality. Code has been updated to no longer set the active state on enable/disable/reset as this did not need to be done as local storage can be updated with just a key name.

File: dist/featureFlags.js [hunk 1]
```diff
@@ -170,18 +185,29 @@ function FeatureFlags($q, featureFlagOverrides, initialFlags) {
...
+
+        changeEnvironmentVal = function(flag, value) {
+            Object.keys(flag.environments).forEach(function(env) {
+                flag.environments[env] = value;
+            });
+        },
+
         enable = function(flag) {
-            flag.active = true;
+            changeEnvironmentVal(flag, true);
```
Comment (phdesign) [inline dist/featureFlags.js]: This doesn't look like it has the right changes in it. I think you'll need to do a dist / release again and check that in? It's weird that he checks in compiled files anyway...

**Comment by phdesign** [comment]
Apart from the generated dist file that needs to be updated, it looks good to me, it has the changes we discussed.

**Comment by KylieTims** [comment]
I never thought to check in anything under the dist folder before, I have now so will merge in this branch

**Pull request merged**